packages/apps/docs/src/docs/chainweb/compact-a-database.md

```diff
@@ -0,0 +1,99 @@
+
+To compact a Chainweb node database:
+
+1. Open a terminal shell on a computer with access to the `chainweb-node` you manage.
+   
+   For example, if you run the node in a Docker container, open a terminal in the container.
+   If you installed chainweb-node from a release binary or built it from source, open a terminal or secure shell on the computer where the binary is installed.
+
+   For information about installing chainweb-node from a release binary or building it from source code, see the [chainweb-node README](https://github.com/kadena-io/chainweb-node#README).
+
+2. Verify that you have access to the `cwtool`  command-line program by running the following command:
+   
+   ```bash
+   cwtool
+   ```
+
+   If you have access to the cwtool, you should see usage information similar to the following:
+
+   ```bash
+   Chainweb Tool
+
+   This executable contains misc commands that have been created for various
+   reasons in the course of Chainweb development. Linking executables is slow and
+   the resulting binaries are large, so it is more efficient in terms of build
+   time, space usage, download time, etc to consolidate them into one binary.
+   
+   Usage: cwtool COMMAND
+   
+   Available options:
+     -h,--help                Show this help text
+   
+   Available commands:
+     ea                       Generate Chainweb genesis blocks and their payloads
+     run-nodes                Run a local cluster of chainweb-node binaries
+     slow-tests               Run slow Chainweb tests
+     tx-list                  List all transactions in a chain starting with the most recent block
+     genconf                  Interactively generate a chainweb-node config
+     header-dump              Dump Block Headers to a JSON array
+     b64                      Command line utlis for Chainweb base64 encode/decode
+     db-checksum              Generate a checksum of all the checkpointer database tables between an inclusive range of blocks.
+     known-graphs             Encode know graphs as JSON values
+     tx-sim                   Simulate tx execution against real pact dbs
+     compact                  Compact pact database
+     pact-diff                Diff the latest state of two pact databases
+     calculate-release        Calculate next service date and block heights for upgrades
+   ```
+
+3. Create a backup copy of your current pact `sqlite` database to save all current state by running a command similar to the following:
+   
+   ```bash
+   cp -r /data/state/chainweb/db/0/sqlite /tmp/sqlite-backup-dd-mm-yyyy
+   ```
+   
+   Creating a backup copy of the database requires you to have more disk space available to store both the active database and the backup until you complete the compaction. 
+   However, creating a backup ensures that the node continues to run uninterrupted while the database is being compacted. 
```

Comment:
The other benefit, which is also very important, is that if something goes wrong, you didn't ruin your only copy.